### PR TITLE
feat: integrate with API for suggest changes

### PR DESCRIPTION
**Pull request by FredrikOseberg**
* Adds API integration for the suggest changes endpoints

## About the changes

Closes # .

### CODE REVIEW

**Review by sjaanus — APPROVED** [review]
LGTM!

File: frontend/src/component/suggestChanges/SuggestedChangesSidebar/SuggestedChangesSidebar.tsx [hunk 1]
```diff
@@ -45,10 +46,22 @@ export const SuggestedChangesSidebar: VFC<ISuggestedChangesSidebarProps> = ({
...
+    const {
+        draft,
+        loading,
+        refetch: refetchSuggestChange,
+    } = useSuggestedChangesDraft(project);
+    const { changeState } = useSuggestChangeApi();
 
-    const onReview = async () => {
-        alert('approve');
+    console.log(draft);
```
Comment (kwasniew) [inline frontend/src/component/suggestChanges/SuggestedChangesSidebar/SuggestedChangesSidebar.tsx:56]: do you want to remove it?

File: frontend/src/hooks/api/actions/useSuggestChangeApi/useSuggestChangeApi.ts [hunk 2]
```diff
@@ -32,8 +33,41 @@ export const useSuggestChangeApi = (project: string) => {
...
+        payload: any
+    ) => {
+        const path = `api/admin/projects/${project}/suggest-changes/${suggestChangeId}/state`;
+        const req = createRequest(path, {
+            method: 'PUT',
+            body: JSON.stringify(payload),
+        });
+        try {
+            const response = await makeRequest(req.caller, req.id);
+            return await response.json();
```
Comment (kwasniew) [inline frontend/src/hooks/api/actions/useSuggestChangeApi/useSuggestChangeApi.ts:48]: await not needed after return

File: frontend/src/component/suggestChanges/SuggestedChangesSidebar/SuggestedChangesSidebar.tsx [hunk 3]
```diff
@@ -45,10 +46,22 @@ export const SuggestedChangesSidebar: VFC<ISuggestedChangesSidebarProps> = ({
...
+
+    const onReview = async (draftId: number) => {
+        try {
+            await changeState(project, draftId, { state: 'In review' });
+            refetchSuggestChange();
```
Comment (kwasniew) [inline frontend/src/component/suggestChanges/SuggestedChangesSidebar/SuggestedChangesSidebar.tsx:61]: refetchSuggestedChanges?

**Review by andreas-unleash — APPROVED** [review]
LG!

**Review by kwasniew — APPROVED** [review]
Minor comments

**Pull request merged**